src/Uri.php

```diff
@@ -281,6 +287,11 @@ public static function removeDotSegments($path)
      */
     public static function resolve(UriInterface $base, $rel)
     {
+        @trigger_error(
```

Comment:
The `@` sign mutes the warning, so that it will never be visible to the implementing developer.